.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js:
 - '6'
 - '8'
 script:
-- ./bin/run-test.sh -c
+- gulp && ./bin/run-test.sh -c
```

Comment:
Good catch.  I've added a `build` script and now `npm run build` is run instead of directly running `gulp`.